components/SiteNavbar.vue

```diff
@@ -1,38 +1,231 @@
 <template>
-  <nav class="flex flex-wrap flex-row-reverse justify-between border-b border-gray-300 px-5 py-3 lg:flex-row">
-    <div class="flex lg:hidden" @click="mobileNavIsActive = !mobileNavIsActive">
-      <button class="flex items-center border border-gray-300 rounded px-4 py-1 text-gray-600 hover:text-black hover:border-black hover:bg-gray-100">
-        <svg class="fill-current h-3 w-3" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
-          <title>Menu</title>
-          <path d="M0 3h20v2H0V3zm0 6h20v2H0V9zm0 6h20v2H0v-2z"/>
+  <nav class="border-b border-gray-300">
+    <div
+      class="
+        mx-auto
+        container
+        flex flex-wrap flex-row-reverse
+        justify-between
+        px-5
+        py-3
+        lg:flex-row
+      "
+    >
+      <div class="flex lg:hidden">
```

Comment:
To space the buttons out I would suggest adding `flex-row space-x-2` to the parent flex div and removing `m-1` from the child buttons. Reduces the number of classes and makes the spacing more consistent.